apps/storefront/docs/patterns/grid-alignment/grid-alignment.mdx

```diff
@@ -15,7 +15,7 @@ route: /patterns/grid-alignment/
 
 ## Guidelines
 
-Using a grid as a starting point for your component implementation will help you arrange your content into structured propositions. You also have to keep in mind that EDS is based on an 8px ratio. This means that you always have to keep a minimum of 8px beetween objects and elements.
+Using a grid as a starting point for your component implementation will help you arrange your content into structured proportions. You also have to keep in mind that EDS is based on an 8px ratio. This means that you always have to keep a minimum of 8px beetween objects and elements.
 
 ### Don’t
 
```

Comment:
- Do not use less than 8px between each components. 
Either say ...between components. 
or ... between each component.